includes/wikia/api/DWDimensionApiController.class.php

```diff
@@ -111,6 +117,43 @@ public function getWikis() {
 		);
 	}
 
+	public function getUsers() {
+		$db = $this->getSharedDbSlave();
+
+		$limit = min( $db->strencode( $this->getRequest()->getVal(
+			'limit', static::LIMIT ) ), static::LIMIT_MAX );
+		$afterUserId = $db->strencode( $this->getRequest()->getVal(
+			'after_user_id', static::DEFAULT_AFTER_USER_ID ) );
+
+		$query = str_replace( '$user_id', $afterUserId, DWDimensionApiControllerSQL::DIMENSION_USERS );
```

Comment:
Query param encoding is missing

Reply:
What do you mean? $db->strencode is called above or is there some other encoding you have in mind? aa or do you mean using getInt instead of getVal?